bouncer/src/repo/repo_controller_internal.cpp.inl

```diff
@@ -38,6 +39,30 @@ RepoController::_RepoControllerImpl::_RepoControllerImpl(
 	{
 		subscribeToLogger(listeners);
 	}
+
+	// set logging directory
+	char* logDir = getenv("REPO_LOG_DIR");
```

Comment:
Can we refactor `getenv` logic into repo_utils and get everywhere to just call that function? so it takes in a envVar to look for and return ` std::string`

It's getting to the point where we are doing a lot of duplicated code 😆